### PR TITLE
Template update nf-core tools v2.8

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Template update nf-core tools v2.8

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/atacseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/atacseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,36 +67,33 @@ On release, automated continuous integration tests run the pipeline on a full-si
...
-   > - If you are using `singularity`, please use the [`nf-core download`](https://nf-co.re/tools/#downloading-pipelines-for-offline-use) command to download images first, before running the pipeline. Setting the [`NXF_SINGULARITY_CACHEDIR` or `singularity.cacheDir`](https://www.nextflow.io/docs/latest/singularity.html?#singularity-docker-hub) Nextflow options enables you to store and re-use the images from a central location for future pipeline runs.
-   > - If you are using `conda`, it is highly recommended to use the [`NXF_CONDA_CACHEDIR` or `conda.cacheDir`](https://www.nextflow.io/docs/latest/conda.html) settings to store the environments in a central location for future pipeline runs.
-
-4. Start running your own analysis!
-
-   ```bash
-   nextflow run nf-core/atacseq --input samplesheet.csv --outdir <OUTDIR> --genome GRCh37 -profile <docker/singularity/podman/shifter/charliecloud/conda/institute>
-   ```
-
-See [usage docs](https://nf-co.re/atacseq/usage) for all of the available options when running the pipeline.
```
Comment (mashehu) [inline README.md:95]: you somehow lost the links to the usage and parameter docs from the template: For more details, please refer to the [usage documentation]([https://nf-co.re/{{](https://nf-co.re/%7B%7B) short_name }}/usage) and the [parameter documentation]([https://nf-co.re/{{](https://nf-co.re/%7B%7B) short_name }}/parameters).

File: modules/local/tss_extract.nf [hunk 2]
```diff
@@ -3,7 +3,7 @@ process TSS_EXTRACT {
     conda (params.enable_conda ? "conda-forge::sed=4.7" : null)
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
-        'ubuntu:20.04' }"
+        'docker.io/ubuntu:20.04' }"
```
Comment (mashehu) [inline modules/local/tss_extract.nf:6]:
```suggestion
        'docker.io/library/ubuntu:20.04' }"
```
see https://github.com/nf-core/modules/pull/3358

File: modules/local/multiqc_custom_peaks.nf [hunk 3]
```diff
@@ -3,7 +3,7 @@ process MULTIQC_CUSTOM_PEAKS {
     conda (params.enable_conda ? "conda-forge::sed=4.7" : null)
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
-        'ubuntu:20.04' }"
+        'docker.io/ubuntu:20.04' }"
```
Comment (mashehu) [inline modules/local/multiqc_custom_peaks.nf:6]:
```suggestion
        'docker.io/library/ubuntu:20.04' }"
```
see https://github.com/nf-core/modules/pull/3358

**Review by mashehu — APPROVED** [review]

File: modules/local/tss_extract.nf (hunk 2 above)
Comment (JoseEspinosa) [reply modules/local/tss_extract.nf:6]: Thanks!

**Pull request merged**